packages/wrangler/src/config/validation.ts

```diff
@@ -1192,8 +1187,13 @@ const validateUnsafeBinding: ValidatorFn = (diagnostics, field, value) => {
     const safeBindings = [
       "plain_text",
       "json",
+      "wasm_module",
+      "data_blob",
+      "text_blob",
       "kv_namespace",
       "durable_object_namespace",
+      "r2_bucket",
```

Comment:
These new "safe" bindings seem unrelated to this PR?

Reply:
added it because it was missed when it was actually added, it simply adds a warning if you use them in unsafe.bindings.